File: output/ChatGPT/codenet/Python/Java/s893778513.java
import java.util.Scanner;
import java.util.ArrayList;
import java.util.List;

public class s893778513 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        
        List<Integer> l = new ArrayList<>();
        for (int i = 0; i < n; i++) {
            l.add(scanner.nextInt());
        }
        
        int b = Collections.max(l);
        int s = 0;
        for (int num : l) {
            s += num;
        }
        
        if (b < s - b) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}

File: output/ChatGPT/codenet/Python/C++/s893778513.cpp
#include <iostream>
#include <vector>
#include <algorithm>

int main() {
    int n;
    std::cin >> n;
    
    std::vector<int> l(n);
    for (int i = 0; i < n; i++) {
        std::cin >> l[i];
    }
    
    int b = *std::max_element(l.begin(), l.end());
    int s = 0;
    for (int num : l) {
        s += num;
    }
    
    if (b < s - b) {
        std::cout << "Yes" << std::endl;
    } else {
        std::cout << "No" << std::endl;
    }
    
    return 0;
}